src/main/java/core/basesyntax/Zoo.java

```diff
@@ -9,6 +9,30 @@
     private static List<Bird> birdHouse = new ArrayList<>();
     private static List<Swimmable> aquarium = new ArrayList<>();
```

Comment:
one list you arew using interface to parametrize method, in others Abstract classes